src/demo/components/TPageHighlighted.tsx

```diff
@@ -0,0 +1,34 @@
+import { TH2 } from "../../common/atoms/THeadings";
+import { Highlighter } from "rc-highlight";
+import PropTypes from "prop-types";
+
+const HighlightedPageView = ({ highlighterText, documentLink, githubLink }) => {
+  return (
+    <div className="rightDocs">
```

Comment:
What is this class name CSS targeting?

Reply:
Just found it in pageLayout.css. You should not use the CSS from another components CSS. component CSS should be modular - only used by its component.

If you want common CSS this should be inside a file that is shared by others.